src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  switch (true) {
+    case days >= 7:
+      return days * price - 50;
+
+    case days >= 3:
+      return days * price - 20;
```

Comment:
GREAT, but let's improve your code)

It's better to create variables for all discounts

https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md

checklist 1.